assets/js/pages/HostDetailsPage/HostDetails.jsx

```diff
@@ -67,6 +69,20 @@ export const buildCidrNotation = (ipAddresses, netmasks) =>
     (address, netmask) => `${address}${netmask ? `/${netmask}` : ''}`
   );
 
+const clusterCanReboot = (cluster) => {
+  if (!cluster) {
+    return true;
+  }
+  switch (get(cluster, 'type')) {
```

Comment:
We have the `isValidClusterType` function in `lib/model/cluster.js`. It could help:
```
... // instead of the swtich
return isValidClusterType(get(cluster, 'type'))
```